test/core/tsi/ssl_transport_security_utils_test.cc

```diff
@@ -429,6 +445,184 @@ INSTANTIATE_TEST_SUITE_P(FrameProtectorUtil, FlowTest,
 
 #endif  // OPENSSL_IS_BORINGSSL
 
+X509_CRL* read_crl(absl::string_view crl_string) {
+  BIO* crl_bio =
+      BIO_new_mem_buf(crl_string.data(), static_cast<int>(crl_string.size()));
+  // Errors on BIO
+  if (crl_bio == nullptr) {
+    return nullptr;
+  }
+  X509_CRL* crl = PEM_read_bio_X509_CRL(crl_bio, nullptr, nullptr, nullptr);
+  BIO_free(crl_bio);
+  return crl;
+}
+
+X509* read_cert(absl::string_view cert_string) {
```

Comment:
Same here RE changing the variable name to `cert_pem`.

Reply:
Done